### ISSUE TITLE
[INFO] Status of mds-core

### ISSUE THREAD

**Issue by mrsimpson** [CityOfLosAngeles/mds-core#268]
Dear OMF runtime producers,

we're currently looking into `mds-core` and evaluate whether to use it as basis for our implementation.
Unfortunately the repository seems to be a bit deserted, the specification is developing much faster than its runtime.
I would therefore appreciate a maintainer (@schnuerle ?) commenting on what the OMF is up to with this implementation. 

At this point in time, we've got the options to
- develop and contribute to `mds-core` (this would of course be some overhead on our end, but if it helps for the greater good, we'd see what we can do ;) 
- just fork it without contributing (we'd go this route if you, the OMF is not aiming at providing a reference implementation on this basis)
- develop something completely new (in Java Spring Boot), with both the options to opensource it or not

What are your thoughts about it, given that you know best about the strategic goals, current  and future challenges of OMF and MDS?

Cheers, 
Oliver

**Comment by marie-x** [comment]
The actively-maintained version of mds-core, which is deployed in Los Angeles, is here:

https://github.com/lacuna-tech/mds-core

**Comment by schnuerle** [comment]
Hi @mrsimpson these are good comments and thoughts. You are right that this repo is behind on its feature and version support. There is a more up to date [version here](https://github.com/lacuna-tech/mds-core) which you may take a look at. 

Core is a product that we took on as a contribution to OMF a few months ago in June. Our hope was that others would start using it and give feedback and pitch in on development [based on this guidance](https://github.com/openmobilityfoundation/mds-core/wiki), so we certainly welcome your offer of help with your first option. To that end, it could be something we revisit and start to discuss as part of the [City Services Working Group](https://github.com/openmobilityfoundation/mobility-data-specification/wiki/MDS-City-Services-Working-Group#mds-core-project). Lately conversation has been focused on MDS, but with us possibly slowing the release pace of MDS, this may be a good time to revisit Core. @karcass what do you think of this and maybe merging the latest Core here with appropriate documentation?

Your option 2 is certainly your choice, though maybe we can see if your comment here has helped kickstart the OMF community to working on Core together instead.

Option 3 is also possible, and could be an option if you find Core to not fit your needs. When complete if you choose to open source it that would be welcome and we can certainly reference it in our [State of Practice](https://github.com/openmobilityfoundation/privacy-committee/blob/main/products/state-of-the-practice.md#data-processing-aggregation-and-anonymization) and [Community Projects](https://www.openmobilityfoundation.org/community-projects/) areas as a start.

**Comment by mrsimpson** [comment]
Thanks a ton, @schnuerle ! This was exactly the guidance & information I was looking for at the moment. The links provided are very valuable and will surely influence our team's decision on how to proceed.

@karcass thanks for chiming in here! Not having looked at your repo in detail (I only set a watch for releases for now ;) ) it's great to see somebody is taking care of bringing this implementation forward!
May I know the reason for your fork being far ahead without merging it back? If we decided to build on top of Core, we'd fork off your implementation, risking OMF/Core to be superseeded in the long run - and I don't think anyone of us liked this.

With respect to this issue: Feel free to close it whenever you feel appropriate. If you e. g. wanted to take this as an anchor for discussing the state of upstream retrofit of `lacuna-tech/mds-core`, I'd much appreciate this openness, but I of course understand if you prefer to discuss this Off-Github ;)

Cheers and keep up the great work, 
Oliver
